docs/dietpi_references.md

```diff
@@ -0,0 +1,23 @@
+# References
+
+At this page, links to general information are referenced.  
+Links to special software packages are located at the end of the according software package description in the "Optimized Software" documentation area.
+
+## General links
+
+- Twitter: <https://twitter.com/dietpi_>
+- Facebook: <https://www.facebook.com/groups/167403007229675>
+
+## General articles
+
+- Raspberry Pi Foundation: [DietPi | Minimal image, highly extensible](https://www.raspberrypi.org/forums/viewtopic.php?t=285096)
+- Hardkernel ODROID: [[OS] DietPi | Minimal image, highly extensible](https://forum.odroid.com/viewtopic.php?t=40392)
+- PC Welt (german language): [`Diet Pi ist erste Wahl für Headlessplatinen`](https://www.pcwelt.de/a/diet-pi-ist-erste-wahl-fuer-headlessplatinen,3452321)
```

Comment:
I wanted to correct it being "DietPi" without space, but I guess it has priority to use the actual video title, right? 😄

Reply:
Yes, I took the original titles.